src/getHumanAge.test.js

```diff
@@ -7,4 +7,54 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array', () => {
+    expect(getHumanAge(16, 16))
+      .toBeInstanceOf(Array);
+  });
+
+  test(`should return an array with two zeros
+    if catAge and dogAge are equal to 0`, () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return an array with two zeros
+    if catAge and dogAge < 15`, () => {
+    expect(getHumanAge(13, 11))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return an array with two ones
+    if catAge and dogAge = 15`, () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test(`should return an array with two ones
+    if catAge and dogAge > 15 but < 24`, () => {
+    expect(getHumanAge(16, 23))
+      .toEqual([1, 1]);
+  });
+
+  test(`should return an array with two twoes
+    if catAge and dogAge = 24`, () => {
+    expect(getHumanAge(24, 24))
+      .toEqual([2, 2]);
+  });
+
+  test(`should return an array with two twoes
+    if catAge and dogAge = 27`, () => {
+    expect(getHumanAge(27, 27))
+      .toEqual([2, 2]);
+  });
+
+  test(`should return an array with converted
+    catAge and dogAge to human age if catAge and dogAge >= 28`, () => {
+    expect(getHumanAge(100, 100))
+      .toEqual([21, 17]);
+
```

Comment:
It's better to split this test case into two separate test cases, one for each unique input, to improve readability and make it easier to debug test failures.